cloudevents/working-drafts/xml-format.md

```diff
@@ -135,19 +135,35 @@ Example:
 
 ### 3.2 Text Data
 
-Text MUST be carried in an element with a defined type of `xs:string`.
-The element MUST NOT contain any child elements.
+Text MUST be carried in an element with a defined type of `xs:string`. The content
+of this element is either:
+
+* Regular XML textual content, or
+* A CDATA section.
+
+Implementations MUST accept both textual representations.
```

Comment:
This currently seems to restrict it to one or the other - I *suspect* a mixture makes sense, e.g.

```xml
<data xsi:type="xs:string">Text before <![CDATA[if X < Y { .. }]]> Text after</data>
```

Something like "The content of this element is text, which may be expressed as any combination of text nodes and CDATA nodes. Implementations MUST accept all combinations."

(Not sure about the "all combinations" it, but we can noodle on that further.)